sys/arm64/arm64/vm_machdep.c

```diff
@@ -209,7 +213,15 @@ cpu_set_upcall(struct thread *td, void (* __capability entry)(void *),
 		tf->tf_x[13] = STACKALIGN((uintcap_t)stack->ss_sp + stack->ss_size);
 	else
 		tf->tf_sp = STACKALIGN((uintcap_t)stack->ss_sp + stack->ss_size);
+
+#if __has_feature(capabilities)
+	if (SV_PROC_FLAG(td->td_proc, SV_CHERI))
+		trapframe_set_elr(tf, (uintcap_t)entry);
```

Comment:
Should we add a comment that the toolchain sets the LSB on entry, so we should not use cheri_capmode() here?

Reply:
We don't actually rely on that any more, in order to make the sigcode cap be C64 we use `cheri_capmode` in `_cheri_capability_build_user_code` for SV_CHERI.